src/web/client/src/components/LiveStream/LiveStream.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { render } from "@testing-library/react";
+import LiveStream from "./LiveStream";
+
+describe("LiveStream component", () => {
+  let mockCollection: object;
+  let mockJsonPromise: any;
+  let mockFetchPromise: any;
```

Comment:
Shouldn't the type be Promise<object> and Promise?

Reply:
I thought so too, but I couldn't find a right typing for this.

Reply:
It gives me this `Generic type 'Promise<T>' requires 1 type argument(s).`

Reply:
what about Promise<any> :)

Reply:
yes